x-pack/plugins/security/public/views/management/edit_role/components/privileges/kibana/space_aware_privilege_section/privilege_space_form.tsx

```diff
@@ -261,10 +260,14 @@ export class PrivilegeSpaceForm extends Component<Props, State> {
               },
             ]}
             hasDividers
-            valueOfSelected={`basePrivilege_${this.getDisplayedBasePrivilege(
-              allowedPrivileges,
-              baseExplanation
-            )}`}
+            valueOfSelected={
+              this.getDisplayedBasePrivilege(allowedPrivileges, baseExplanation)
```

Comment:
Probably don't want to double-invoke `this.getDisplayedBasePrivilege` -
1. it performs a couple loops; these are (currently and likely always) against small arrays, but there is a chance they could grow, and/or this function's complexity increases for other reasons
2. readability & maintainability; it takes a couple moments (at least for me) to realize what this is doing, and it would be very easy for someone in the future to modify one of the calls without the other.

Unfortunately, there isn't a clean one-liner to solve this; a variable has to be created to hold the result and re-use the variable. I can think of a couple inline solutions but none of them are clean. I think the best way to solve is to modify the two `return`s in `getDisplayedBasePrivilege` to have this logic, allowing the JSX to remain simple.

`getDisplayedBasePrivilege` is a private member on the class and only called in this one location, so updating it's return value is safe.